testing-guidance/workflow-tests/assetpipeline/assetpipeline-assetbundling-workflow-tests.md

```diff
@@ -0,0 +1,96 @@
+| Launch the Editor and load a level.   | -  Create a new level<br> -  Load an existing level                                                                                                    | -  There should be no console errors<br> -  There should be no Asset Validation warings when neither seed mode nor bundle mode are enabled                                                                                              |
+| Enable Seed mode and enter game mode. | -  Have no assets in the level<br> -  Have default assets in the level<br> -  Have custom assets in the level<br> -  Add new assets                    | -  There should be no assets reported missing when no assets are loaded<br> -  Assets should be reported as missing when default or custom assets are preset<br> -  Excluded assets, such as shaders, should not be reported as missing |
+| Add seedpath.                         | -  Add seedpath for level<br> -  Add seedpath for individual assets present in level<br> -  Add seedpath for assets not currently present in the level | -  Added assets and their dependencies should not report as missing<br> -  Added seedpaths not present in level should not report as missing                                                                                            |
+| Remove a seedpath.                    | -  Remove seedpath for default assets<br> -  Remove seedpath for custom assets<br> -  Remove seedpath for asset not present in level                   | -  Removed seedpath of asset not present in level should not report as missing<br> -  Removed seedpath of asset present in level should report as missing                                                                               |
+| Create a seedlist.                    | -  Include assets from the previous seedpath                                                                                                           |                                                                                                                                                                                                                                         |
+| Add a seedlist.                       |                                                                                                                                                        | -  Added seedlist assets present in level should no longer report as missing <br> -  Added seedlist assets not present in the level should not report as missing                                                                        |
```

Comment:
Some of these cells are empty. Is that intended?

Reply:
Yeah, if it's in the request section, there's just no additional requests for the workflows. 
If it's in the Things To Watch Out For Section, then it's because what could go wrong in this portion isn't directly tied to what we are testing, and as such only the general things to watch out for should apply. 
The general list of these is in a section above.